spec/controllers/idv/please_call_controller_spec.rb

```diff
@@ -93,6 +93,22 @@
       expect(response).to redirect_to(idv_not_verified_url)
     end
 
+    context 'user fails ipp' do
+      let!(:enrollment) { create(:in_person_enrollment, :failed, user: user, profile: profile) }
+
+      it 'returns true from in_person_prevent_fraud_redirection' do
+        expect(subject.in_person_prevent_fraud_redirection?).to eq(true)
+      end
+
+      it 'does not redirect a user who has been fraud rejected' do
+        profile.reject_for_fraud(notify_user: false)
+
+        get :show
+
+        expect(response).not_to redirect_to(idv_not_verified_url)
```

Comment:
The directions in your testing plan say to `Login again, and attempt to navigate past the /account page`.
I don't see that you are doing that here. I added `expect(response).to redirect_to(account_url)` and you are only still on the account page. Do you think we should mimic moving past the account page and only then confirming the url is either the next page (when ipp failed/threatmetrix passed) or is the "not verified url" (when ipp failed/threatmetrix failed)? I think you will see a lot passing tests with not redirect to so this test could look like it is passing even though it is not.

Reply:
Good call out and thanks for checking that! So we could check that, I think that slightly deviates from what a user would experience with a service provider in place if they wanted to redo the IPP flow, but I'm not totally sure. 

I _think_ this is sufficient since we really want to make sure that the `fraud_review_concern` is doing what it's supposed to as a before action. And that get checked in the rejection step. Anything after that is not totally within the scope of this ticket.

Reply:
Jack and I discussed this further.  I am going to resolve thread as integration tests will be captured in a diff ticket.